src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -193,6 +193,12 @@ class CommandRequestImpl : public CommandImpl,
   bool CheckAllowedParameters();
 
   /**
+   * @brief Checks HMI capabilities for specified button support
+   * @param button Button to check
+   */
+  bool CheckHMICapabilities(mobile_apis::ButtonName::eType button) const;
```

Comment:
@dev-gh added in https://github.com/smartdevicelink/sdl_core/pull/1621/commits/e5fb4510b64635d824bbb48a205b7c3e4ed0e92c